### PR TITLE
remove `editor.exec` from example code

### PR DESCRIPTION
**Pull request by Louis-Tian**
replace the `editor.exec` call with the new command method.

#### Is this adding or improving a _feature_ or fixing a _bug_?
bugfix? - updating the documentation.

### CODE REVIEW

File: docs/walkthroughs/02-adding-event-handlers.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ const App = () => {
             // Prevent the ampersand character from being inserted.
             event.preventDefault()
             // Execute a command to insert text when the event occurs.
-            editor.exec({ type: 'insert_text', text: 'and' })
+            Editor.insertText(editor, 'and')
```
Comment (shankardevy) [inline docs/walkthroughs/02-adding-event-handlers.md]:
```suggestion
            editor.insertText(editor, 'and')
```

Comment (shankardevy) [inline docs/walkthroughs/02-adding-event-handlers.md]: where is `Editor` coming from?

Comment (Louis-Tian) [reply docs/walkthroughs/02-adding-event-handlers.md]: https://github.com/ianstormtaylor/slate/blob/8cd9b793528eb76a4a76940e022854cf33419bc4/site/examples/richtext.js#L4

**Comment by rockettomatooo** [comment]
Possible duplicate of #3451 ?

**Comment by CameronAckermanSEL** [comment]
Hey @Louis-Tian , thanks for the fix. It looks like this error has already been fixed on this document so I'm going to close this PR. If you find that the existing change is still incorrect, let me know and I'll reopen it.

Thanks!

**Pull request closed without merging**